src/jumpstart/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { getJumpstartContractAddress } from 'src/jumpstart/selectors'
```

Comment:
i think we are trying to not create more general "utils" files because they become a dumping ground for random functions (sometimes poorly thought out abstractions). perhaps we can rename this file to something like "transactionClassificationHelpers" or something? or perhaps we can add to the mess in the transactions folder, there is already a utils file there that this function might make sense in :)